plugins/single_plugins/resize.cpp

```diff
@@ -104,6 +104,20 @@ class wayfire_resize : public wf::plugin_interface_t
         output->connect_signal("view-disappeared", &view_destroyed);
     }
 
+    bool activate(bool keep) {
```

Comment:
I'd like a longer name for the parameter name, `keep` does not immediately tell the reader what this parameter is all about.

Reply:
I've changed it to `should_perserve_aspect`.